apps/console/src/features/core/hooks/use-routes.tsx

```diff
@@ -155,14 +156,16 @@ const useRoutes = (): useRoutesInterface => {
                 && AppConstants.ORGANIZATION_ENABLED_ROUTES;
         }
 
+        const checkConsoleScopes: boolean = !(legacyModeConfigs?.consoleFeatureScopeCheck === false);
```

Comment:
Do we really need this variable here?

We can pass `legacyModeConfigs.consoleFeatureScopeCheck` directly to the function.

Reply:
checkConsoleScope option should be disabled only if legacyModeConfigs?.consoleFeatureScopeCheck is explicitly set to false, otherwise in every case console scope check should be true. I can just pass the `!(legacyModeConfigs?.consoleFeatureScopeCheck === false)` check without using variable. Will do that.

Reply:
Ack.

Let's keep this variable then.

Reply:
Comment is added before the variable declaration.